src/interface/src/app/types/region.types.ts

```diff
@@ -20,15 +23,22 @@ const regions: Region[] = [
   Region.NORTHERN_CALIFORNIA,
 ]
 
-const availableRegions = new Set([
-  Region.SIERRA_NEVADA,
-])
+const availableRegions = new 
+Set([Region.SIERRA_NEVADA,
+  features.show_socal ? Region.SOUTHERN_CALIFORNIA : null])
+
+const rawConditionRegions = new Set([Region.SIERRA_NEVADA, Region.SOUTHERN_CALIFORNIA]);
```

Comment:
My main thought here is... could we drive appearances of the data in the json instead of here.  I'm not entirely sold on the alternative either.  Ok to include as is, but ack that we may need to have a flag-driven field somewhere if we are to introduce translated SoCal after launching publicly raw SoCal.

Reply:
Having it in the json feels a lot cleaner to me — I'll look into that. Thanks!